tcptracer-bpf.c

```diff
@@ -666,7 +725,7 @@ int kretprobe__tcp_v4_connect(struct pt_regs *ctx) {
 	// We should figure out offsets if they're not already figured out
 	are_offsets_ready_v4(status, skp, pid);
 
-	return 0;
```

Comment:
Perhaps just add the reasoning you explained here as a comment for creating the tcp record instead of returning 0.

Reply:
I added the comment to the create_tcp_record function. That seemed most logical